### ISSUE TITLE
Possible support to display realtime result of script on admin page

### ISSUE THREAD

**Issue by shinebayar-g** [bugy/script-server#170]
In admin panel, if you go to **running** script it shows script output. But it's not realtime. If I refresh the page, it shows updated outputs. If it shows that output without refreshing the page, it would be awesome. What do you think? @bugy

**Comment by bugy** [comment]
Hi @shinebayar-g, I think it would make sense to show realtime result.
But to be honest considering the list of other tickets, I would do this one with the lower priority.

**Comment by shinebayar-g** [comment]
Indeed, there are tons of issues that needs to be done beforehand. And I can't help but create more issues..
